composer.json

```diff
@@ -7,11 +7,11 @@
     "authors": [
         {
             "name": "PHP-FIG",
-            "homepage": "http://www.php-fig.org/"
+            "homepage": "https://www.php-fig.org/"
         }
     ],
     "require": {
-        "php": ">=5.3.0"
+        "php": "^7.2.0 || ^8.0.0"
```

Comment:
As with the first PR, let's go straight to 8.  There are no supported 7.x versions anymore.

Reply:
I disagree.

Ubuntu, RHEL 7, and Debian 11 are all pinned to PHP 7.4 still, and these are all LTS editions. It makes sense to allow them to be able to update to the 1.1 release.